assignments/week_5/week_5_exercises/exercise_5_1/exercise_5_1.md

```diff
@@ -0,0 +1,40 @@
+# Exercise 5.1
+This exercise covers how to create a launch file for node.
+
+# Exercise Objective
+Create a `.launch` file which starts the `printing_node`, which is already created for you.
+
+A launch file structured with XML (like HTML): it is composed of tags, which have attributes and 
```

Comment:
Can we link to a resource with the basic syntax, then I would also explicitly explain how to comment out some lines <!-- -->